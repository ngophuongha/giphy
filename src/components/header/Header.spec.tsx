import { render } from "../../../test";
import { Header } from "./Header.styled";

describe("Styled Header", () => {
  it("should match snapshot", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
