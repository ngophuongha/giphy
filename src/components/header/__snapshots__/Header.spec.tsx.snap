// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Header should match snapshot 1`] = `
.c0 {
  color: #000000;
}

<div>
  <header
    class="c0"
  />
</div>
`;
